src/utilities/calculateDailyEarnings.ts

```diff
@@ -0,0 +1,6 @@
+import BigNumber from 'bignumber.js';
+
+const calculateDailyEarningsCents = (yearlyEarningsCents: BigNumber) =>
+  +yearlyEarningsCents.dividedBy(365).toFixed(0);
```

Comment:
I would return the `BigNumber` directly, so it can be directly reused for calculations.